testing_and_setup/compass/utility_scripts/LIGHTparticles/make_particle_file.py

```diff
@@ -524,11 +583,13 @@ def build_particle_file(f_init, f_name, f_decomp, types, spatialfilter, buoySurf
     if not os.path.exists(args.graph):
         raise OSError('Graph file {} not found.'.format(args.graph))
 
+    assert args.types in typelist, 'Selected particle type is not correct!'
```

Comment:
This currently breaks even if you submit an appropriate list like 'passive','surface'. It should be changed to something like:

`assert set(args.types.split(',')).issubset(typelist), 'Selected particle type is not correct!'`

Reply:
Should be resolved now, thanks!